Bottom: 42ca3ead6677b544ea2c1f680d460bf2792a1abe
Top:    addfa57f1218d7719b0785fd5ad73ffd33f6d5ca
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-07 11:44:09 +0200

Add a calculator controller object.


---

diff --git a/src/modules/calculator/CMakeLists.txt b/src/modules/calculator/CMakeLists.txt
index 4bc9837..e3e7585 100644
--- a/src/modules/calculator/CMakeLists.txt
+++ b/src/modules/calculator/CMakeLists.txt
@@ -7,11 +7,19 @@ target_compile_options(calculator
 	PRIVATE
 		${module_compile_options}
 )
+target_link_libraries(calculator
+	PRIVATE
+		math
+)
+add_module(calculator.controller controller.cpp)
 add_module(calculator.model model.cpp)
 add_module(calculator.value value.cpp)
 add_dependencies(calculator.model.pcm
 	calculator.value.pcm
 )
-add_dependencies(calculator
+add_dependencies(calculator.controller.pcm
 	calculator.model.pcm
 )
+add_dependencies(calculator
+	calculator.controller.pcm
+)
